src/settings/v2/components/QASettings.tsx

```diff
@@ -186,35 +158,58 @@ const QASettings: React.FC<QASettingsProps> = ({ indexVaultToVectorStore }) => {
 
           {/* Exclusions */}
           <SettingItem
-            type="textarea"
+            type="custom"
             title="Exclusions"
             description={
               <>
                 <p>
-                  Comma separated list of paths, tags, note titles or file extension will be
-                  excluded from the indexing process. e.g. folder1, folder1/folder2, #tag1, #tag2,
-                  [[note1]], [[note2]], *.jpg, *.excallidraw.md etc,
+                  Exclude folders, tags, note titles or file extensions from being indexed.
+                  Previously indexed files will remain until a force re-index is performed.
                 </p>
-                <em>
-                  NOTE: Tags must be in the note properties, not the note content. Files which were
-                  previously indexed will remain in the index unless you force re-index.
-                </em>
               </>
             }
-            value={settings.qaExclusions}
-            onChange={(value) => updateSetting("qaExclusions", value)}
-            placeholder="folder1, folder1/folder2, #tag1, #tag2, [[note1]], [[note2]], *.jpg, *.excallidraw.md"
-          />
+          >
+            <Button
+              variant="secondary"
+              onClick={() =>
+                new PatternMatchingModal(
+                  app,
+                  (value) => updateSetting("qaExclusions", value),
+                  settings.qaExclusions,
+                  "Manage Exclusions"
+                ).open()
+              }
+            >
+              Manage
+            </Button>
+          </SettingItem>
 
           {/* Inclusions */}
           <SettingItem
-            type="textarea"
+            type="custom"
             title="Inclusions"
-            description="When specified, ONLY these paths, tags, or note titles will be indexed (comma separated). Takes precedence over exclusions. Files which were previously indexed will remain in the index unless you force re-index. Format: folder1, folder1/folder2, #tag1, #tag2, [[note1]], [[note2]]"
```

Comment:
The original message says inclusion takes precedence over exclusions, but I think it's the opposite. For example, if foo/ folder is in both inclusions and exclusions, foo/ will be excluded.